### ISSUE TITLE
Rule Change: make `spaced-comment` check for spaces after `*`

### ISSUE THREAD

**Issue by thislooksfun** [eslint/eslint#16743]
### What rule do you want to change?

spaced-comment

### What change to do you want to make?

Generate more warnings

### How do you think the change should be implemented?

A new option

### Example code

```js
/*
 * This line is properly spaced
 *But this one is not!
 */
```

### What does the rule currently do for this code?

Currently it says that code is A-OK.

### What will the rule do after it's changed?

Complain about the lack of space between the `*` and `B` on line 3.

### Participation

- [ ] I am willing to submit a pull request to implement this change.

**Comment by kecrily** [comment]
Successfully reproduced. [Check it out at playground.](https://eslint.org/play/#eyJ0ZXh0IjoiLyogZXNsaW50IHNwYWNlZC1jb21tZW50OiBbXCJlcnJvclwiLCBcImFsd2F5c1wiXSAqL1xuXG4vKlxuICogVGhpcyBsaW5lIGlzIHByb3Blcmx5IHNwYWNlZFxuICpCdXQgdGhpcyBvbmUgaXMgbm90IVxuICovXG5cbi8qeCovIiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjoibGF0ZXN0Iiwic291cmNlVHlwZSI6InNjcmlwdCIsImVjbWFGZWF0dXJlcyI6e319LCJydWxlcyI6e30sImVudiI6eyJlczYiOnRydWV9fX0=)

**Comment by mdjermanovic** [comment]
Hi @thislooksfun, thanks for the issue!

Per our rules policies, [stylistic rules are frozen](https://github.com/eslint/eslint#stylistic-rule-updates), so we are not adding new options to the `spaced-comment` rule. You can always copy the existing rule into a custom rule and make it behave the way you want.
